air_flags/rollout.py

```diff
@@ -0,0 +1,128 @@
+import random
+from dataclasses import dataclass
+from datetime import datetime
+from typing import Optional
+
+ROLLOUT_VALUES = [True, False]
+DATE_FORMAT = "%Y-%m-%d"
+CANARY_STRATEGY = "canary"
+SCHEDULED_STRATEGY = "scheduled"
+PROGRESSIVE_STRATEGY = "progressive"
+ROLLOUT_STRATEGIES = [
+    CANARY_STRATEGY,
+    SCHEDULED_STRATEGY,
+    PROGRESSIVE_STRATEGY,
+]
+MIN_PERCENTAGE = 1
```

Comment:
Should we permit 0 `MIN_PERCENTAGE` to start the rollout the next day to deploy?